tests/Component/Basket/BasketElementTest.php

```diff
@@ -276,9 +276,10 @@ public function testGettersSetters(): void
         $this->assertSame($product, $basketElement->getProduct());
 
         // Tests setProductId
+        $basketElement->setProductId('42');
+        $this->assertSame(42, $basketElement->getProductId());
```

Comment:
Why twice? It’s the same in the next lines

Reply:
It's not the same test, here i want to test that if a string of quantity is set it must be return be work as well to an integer. I experienced issue from my side.